src/app/data/components/record-list/record-list.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Attributes } from 'src/app/base/components/record-display/record-display.helper';
+import { ColumnData, RecordListOptions } from './record-list-helper';
+
+@Component({
+  selector: 'kern-record-list',
+  templateUrl: './record-list.component.html',
+  styleUrls: ['./record-list.component.scss']
+})
+export class RecordListComponent implements OnInit {
+
+  @Input() recordList: any[];
+  @Input() attributes: Attributes;
+  @Input() recordListOptions: RecordListOptions;
+
+  @Output() indexHuddleData = new EventEmitter<number>();
+  @Output() similarSearch = new EventEmitter();
+
+  columnsData: ColumnData[];
+
+  constructor() { }
+
+  ngOnInit(): void {
+    this.columnsData = this.prepareColumnsData();
+  }
+
+
+  storePreliminaryRecordIds(index: number) {
+    this.indexHuddleData.emit(index);
+  }
+
+  requestSimilarSearch() {
+    this.similarSearch.emit();
+  }
+
+  prepareColumnsData() {
+    const columnsData = [];
+    columnsData.push({
+      field: 'type',
+      displayName: 'Type',
+      order: 1
+    });
+    columnsData.push({
+      field: 'task',
+      displayName: 'Task',
+      order: 2
+    });
+    columnsData.push({
+      field: 'label',
+      displayName: 'Label',
+      order: 3,
+    });
+    columnsData.push({
+      field: 'amount',
+      displayName: 'Amount',
+      order: 4
+    });
+    columnsData.push({
+      field: 'confidenceAvg',
+      displayName: 'Avg.confidence',
+      order: 5
+    });
+    return columnsData;
+  }
```

Comment:
if this is fixed anyway why dont be use a constant reference which is initiialized from the get go?
something like this outside the class: 
```
const DATA_BROWSER_TABLE_COLUMN_HEADERS:ColumnData[] = [
  { field: 'type', displayName: 'Type', order: 1 },
  { field: 'task', displayName: 'Task', order: 2 },
  { field: 'amount', displayName: 'Amount', order: 4 },
  { field: 'confidenceAvg', displayName: 'Avg.confidence', order: 5 }
]
```

and then just reference it in the onInit.

With this we wouldn't need to recrate it every time the browser is opened